Fix deleted users follow in private participatory spaces (#12878)
decidim/decidim: RELEASE_NOTES.md

These are one time actions that need to be done after the code is updated in the production database.

To delete the follows of ex private users of non transparent assemblies or processes, run from development_app

```console

> bundle exec rake decidim:upgrade:fix_deleted_private_follows

Reviewer: I think the "modern" and recommended way of running rake tasks is through the "bin/rails" binstub 
```suggestion
bin/rails decidim:upgrade:fix_deleted_private_follows
```

You can read more about this change on PR [#15390](https://github.com/decidim/decidim/pull/15390).

> ### 3.. Remove the follows of ex private users

Reviewer: ```suggestion
### 3.2. Remove the follows of former private users
```

To delete the follows of ex private users of non transparent assemblies or processes, run

```console

> ./bin/rails decidim:upgrade:fix_deleted_private_follows

Reviewer: ```suggestion
bin/rails decidim:upgrade:fix_deleted_private_follows
```
